feddlib/amr/tests/stokesAdaptive/main.cpp

```diff
@@ -677,6 +678,13 @@ int main(int argc, char *argv[]) {
 		    else
 		        domainVelocity = domainPressure;
 			
+			domainVelocity->exportNodeFlags();
```

Comment:
domainVelocity->exportNodeFlags();
			domainPressure->preProcessMesh(true,true);
			domainVelocity->preProcessMesh(true,true);

   			domainVelocity->exportSurfaceNormals("domain");
    		domainVelocity->exportElementOrientation("domain");

Also müssen die Funktionen jetzt aufgerufen werden damit die mesh orientierung richtig ist damit man danach immer die korrekten normalen berechnet??

Reply:
Genau, also preprocess muss aufgerufen werden. In stokesAdaptiv nur in jedem Verfeinerungsschritt, weil sich das hier mit dem Refinement immer ändern kann.